ansible/inventory/group_vars/dns.yml

```
@@ -1,4 +1,4 @@
 dns_static_records:
   - { hostname: "gateway", ip: "10.0.10.1" }
   - { hostname: "argocd", ip: "10.0.23.21" }
-  - { hostname: "pvc01", ip: "10.0.31.2" }
\ No newline at end of file
+  - { hostname: "pvc01", ip: "10.0.31.2" }
```

YAMLlint (1.37.1)

[error] too many spaces inside braces

(braces)